### ISSUE TITLE
Don't hide the player while scrolling through comments, recommendations and description

### ISSUE THREAD

**Issue by surajkareppagol** [TeamNewPipe/NewPipe#10997]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

- The player gets cut off (or moves out of the display area) when scrolling through comments, recommended videos and description.

### **Before scrolling**
![Before Scrolling](https://github.com/TeamNewPipe/NewPipe/assets/144083304/eaaf23b7-43d0-45c8-8123-0efb7f12edb5)

### **After scrolling**
![After Scrolling](https://github.com/TeamNewPipe/NewPipe/assets/144083304/91ef783d-7b84-427c-be0c-ff800ca5191f)

- It would be good if the player position is fixed.

### **Similar to YouTube**
![Youtube](https://github.com/TeamNewPipe/NewPipe/assets/144083304/1e48f11f-8a60-49a6-b10d-dc630b0527ae)

### Why do you want this feature?

It is quite annoying to **miss the video content** while **scrolling down and reading comments**, need to seek backward to catch up some things.

### Additional information

**Feature Request:** Fixed **_player_** position.

**Comment by somos61** [comment]
use popup instead of mini player while scrolling through comments, recommended videos and description

**Comment by surajkareppagol** [comment]
Yeah, i have been using **popup**, but it would be good if the position was fixed natively in the app.

But thanks.
